Sources/StreamChatUI/Composer/ComposerVC.swift

```diff
@@ -432,8 +476,12 @@ open class ComposerVC: _ViewController,
         } else {
             createNewMessage(text: text)
         }
-
+        
         content.clear()
+        
+        if let cooldownDuration = channelController?.channel?.cooldownDuration {
+            handleCooldownTimer(with: cooldownDuration)
+        }
```

Comment:
Wouldn't this work as well? 
```suggestion
        checkChannelCooldown()
```